spec/support/be_executable_matcher.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+RSpec::Matchers.define :be_executable do
+  match do |filename|
+    File.new(File.join(project_path, filename)).stat.executable?
```

Comment:
Is this the same thing?

```suggestion
    File.stat(File.join(project_path, filename)).executable?
```

Reply:
Word.